me.acristoffers.Lachesis.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=Lachesis
+Comment=Moirai supervisor
+Categories=Science;Engineering;
+
+Icon=me.acristoffers.Lachesis
+Exec=/usr/bin/flatpak --nosocket=fallback-x11 --socket=x11 run me.acristoffers.Lachesis
```

Comment:
this is wrong. it should just be the executable name, maybe with command line argument substitution

Reply:
Wait, what? If I just put lachesis there it doesn't work. Or should it be the run.sh script? Dropbox'es desktop file also does it like that.

Reply:
it should be the same than a in `command`.